PySimpleGUI.py

```diff
@@ -7439,6 +7443,7 @@ def main():
                     font=('Helvetica', 13), min_size=(500, 500), resizable=True,
                     right_click_menu=['&Right', ['Right', '!&Click', '&Menu', 'E&xit', 'Properties']],
                     ).Layout(layout).Finalize()
+    window.SetMaxSize(size=(1000, 0))
```

Comment:
If I instead call:
`    window.SetMaxSize(size=(200, 200))
`
I am unable to resize the window.  Why does it take away the ability to resize?

Reply:
You set `min_size=(500, 500)` and `window.SetMaxSize(size=(200, 200))`. Obviously, max_size must be greater than min_size.